assets/javascripts/discourse/templates/components/emoji-value-list.hbs

```diff
@@ -0,0 +1,32 @@
+{{#if collection}}
+  <div class="values">
+    {{#each collection as |value index|}}
+      <div class="value" data-index={{index}}>
+        {{d-button action=(action "removeValue")
+                   actionParam=value
+                   icon="times"
+                   class="remove-value-btn btn-small"}}
+        {{input value=value class="value-input" focus-out=(action "changeValue" index)}}
+        {{#if index}}
```

Comment:
@jjaffeux I am confused here, How to check whether this is the last element of the array or not in handlebars?

Reply:
I got the solution😅